src/execution_tree/primitives/console_output.cpp

```diff
@@ -91,11 +94,6 @@ namespace phylanx { namespace execution_tree { namespace primitives
     hpx::future<primitive_result_type> console_output::eval(
         std::vector<primitive_argument_type> const& args) const
     {
-        if (operands_.empty())
-        {
-            return std::make_shared<detail::console_output>()->eval(args, noargs);
-        }
-
```

Comment:
Why have you removed this code?

Reply:
In Python, if you do a print statement with no args, you should still print newline. This is for compatibility.

Reply:
Ok, this is a bug in the primitive, then (I created a ticket, #159). Please revert this change as it would possibly break lazy argument evaluation.

Reply:
@stevenrbrandt as assumed, this was a bug in the implementation of the actual output operation. Please see #160 for a proper fix.

Reply:
Please remove this change from the PR as the problem you were referring to has already been fixed on master.